hexrd/ui/image_stack_dialog.py

```diff
@@ -359,37 +361,44 @@ def get_omega_values(self, num_files):
                         steps = max_total_frames % steps
                     omega.append([start, stop, steps])
                     start = stop
-            omega = np.array(omega)
         elif wedges and num_files == 1:
             # single images need a single wedge, even if discontinuous
             nsteps = sum(np.array(wedges)[:, 2])
-            omega = np.array([[wedges[0][0], wedges[-1][1], nsteps]])
+            omega = [[wedges[0][0], wedges[-1][1], nsteps]]
         else:
             # user did not select file or enter wedges
             delta = MAXIMUM_OMEGA_RANGE / num_files
             omega = np.linspace(
                 [0, 0 + delta],
                 [MAXIMUM_OMEGA_RANGE - delta, MAXIMUM_OMEGA_RANGE],
-                num_files)
+                num_files).astype('uint16')
+            omega = [[b, e, self.frames_per_image] for [b, e] in omega]
+            if max_total := self.state['max_total_frames']:
```

Comment:
So this `if` statement is just checking that the `max_total` is not set to zero, right?